tox.ini

```diff
@@ -41,6 +41,11 @@ commands =
     lint: yamllint -s test/ molecule/
 whitelist_externals =
     find
+# sitepackages=true <- template module <- libselinux bindings which cannot be
```

Comment:
Could you please style it in a form
```ini
# This is needed because:
# 1) ...
# 2) ...
# 3) ...
# 
# Side-effects are:
# 1) ...
# 2) ...
# 3) ...
```